appveyor.yml

```diff
@@ -11,7 +11,7 @@ platform:
 environment:
   matrix:
     - TOOLCHAIN_VERSION: 14.0
-      RUST: 1.26.0
+      RUST: stable
```

Comment:
Capitalize AppVeyor in the commit message.

Reorder this commit to come after the switch to `rustup`.

Include the text "I agree to license my contributions to each file under the terms given at the top of each file I changed." in each commit as described in the Contributing section of the README.